sdk/communication/communication-short-codes/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - Exposed top/skip params to listUSProgramBriefs
+- Updating API signature, numberType => value
```

Comment:
This should be listed as a breaking change, as opposed to feature added.

Reply:
fixed